modules/identity-shiro/src/main/java/org/opensearch/identity/shiro/OpenSearchSecurityManager.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.identity.shiro;
+
+import org.apache.shiro.subject.Subject;
+import org.apache.shiro.mgt.DefaultSecurityManager;
+import org.apache.shiro.mgt.SubjectDAO;
+import org.opensearch.identity.shiro.realm.InternalRealm;
+
+/**
+ * OpenSearch specific security manager implementation
+ *
+ * @opensearch.experimental
+ */
+public class OpenSearchSecurityManager extends DefaultSecurityManager {
```

Comment:
Should this be `ShiroSecurityManager` to be consistent with other classes in this namespace and distinguish with other implementations?